cms/static/js/main.js

```diff
@@ -0,0 +1,71 @@
+(function(AjaxPrefix) {
+    'use strict';
+    define(['domReady', 'jquery', 'underscore.string', 'backbone', 'gettext',
+            'common/js/components/views/feedback_notification', 'coffee/src/ajax_prefix',
+            'jquery.cookie'],
+    function(domReady, $, str, Backbone, gettext, NotificationView) {
+        var main;
+        main = function() {
+            AjaxPrefix.addAjaxPrefix(jQuery, function() {
+                return $("meta[name='path_prefix']").attr('content');
+            });
+            window.CMS = window.CMS || {};
+            window.CMS.URL = window.CMS.URL || {};
+            window.onTouchBasedDevice = function() {
+                return navigator.userAgent.match(/iPhone|iPod|iPad|Android/i);
+            };
+            _.extend(window.CMS, Backbone.Events);
+            Backbone.emulateHTTP = true;
+            $.ajaxSetup({
+                headers: {
+                    'X-CSRFToken': $.cookie('csrftoken')
+                },
+                dataType: 'json'
+            });
+            $(document).ajaxError(function(event, jqXHR, ajaxSettings) {
+                var message, msg;
+                if (ajaxSettings.notifyOnError === false) {
+                    return;
+                }
+                if (jqXHR.responseText) {
+                    try {
+                        message = JSON.parse(jqXHR.responseText).error;
+                    } catch (error) {
+                        message = str.truncate(jqXHR.responseText, 300);
+                    }
+                } else {
+                    message = gettext('This may be happening because of an error with our server' +
```

Comment:
Unfortunately ``gettext`` calls cannot be split across lines with the Transifex scraper we have. This needs to be one long line with a comment to disable the warning.